presto-benchto-benchmarks/src/test/resources/sql/presto/tpcds/q34.plan.txt

```diff
@@ -16,10 +16,10 @@ remote exchange (GATHER, SINGLE, [])
                                                         scan store
                                             local exchange (GATHER, SINGLE, [])
                                                 remote exchange (REPLICATE, BROADCAST, [])
-                                                    scan date_dim
```

Comment:
Why did plans change?

Reply:
I'm not sure. Will run `EXPLAIN ANALYZE` for the above query with both the changes and post it here.